src/createNumbersGenerator.js

```diff
@@ -29,8 +29,14 @@
  * @param {number} min - (optional)
  * @param {number} max - (optional)
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(numbers, min = Math.log(0), max = Infinity) {
```

Comment:
It's not necessary to pass default arguments, cause u will compare numbers with `undefined`, and it always returns `false`

Reply:
not quite so, in my comparison both operands should return `true`, in a check with a `false` filter will not work